sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/QueryPlan.scala

```diff
@@ -648,6 +702,8 @@ abstract class QueryPlan[PlanType <: QueryPlan[PlanType]]
 object QueryPlan extends PredicateHelper {
   val OP_ID_TAG = TreeNodeTag[Int]("operatorId")
   val CODEGEN_ID_TAG = new TreeNodeTag[Int]("wholeStageCodegenId")
+  val EXPLAIN_PLAN_INFO_SIMPLE = new TreeNodeTag[String]("explainPlanInfoSimple")
+  val EXPLAIN_PLAN_INFO = new TreeNodeTag[String]("explainPlanInfo")
```

Comment:
Could you add some comments about the difference or (definition) for `explainPlanInfoSimple` and `explainPlanInfo`?

Reply:
Added better comments explaining the reason for the two methods.